app/models/news_item.rb

```diff
@@ -1,7 +1,10 @@
 class NewsItem < ActiveRecord::Base
   extend Enumerize
-  enumerize :state, in: [:draft, :archived, :published]
+  AVAILABLE_STATES = %w(draft archived published)
+  enumerize :state, in: AVAILABLE_STATES
   scope :published, -> { where(state: :published).order(:published_at) }
-  validates :published_at, presence: true, if: -> { state.published? } 
-  validates_presence_of :title, :state
+  validates :published_at, presence: true, if: -> { state.try(:published?) }
+  validates :title, presence: true, length: {maximum: 255}
+  validates :state, presence: true, inclusion: {in: AVAILABLE_STATES, message: "must be one of the following: #{AVAILABLE_STATES.join(', ')}"}
```

Comment:
Line is too long. [142/80]<br>Space inside { missing.<br>Space inside } missing.